wait-for-it.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: wait-for-it
+  version: 0_git200823
```

Comment:
What's does this version `0_git200823` refer to?  I'm wondering how we keep this up to date.  The automation we currently have is for github release versions but it could in the future handle git shas?  For example we used the short git sha for the version
```
package:
  version: 81b13
```
and then in the fetch later we had
```
https://raw.githubusercontent.com/vishnubob/wait-for-it/${{package.version}}/wait-for-it.sh
```
Probably not a blocker but am trying to work out how we might keep this up to date.

Reply:
wait-for-it doesn't have any GitHub Releases, and it looks like there hasn't been a commit in more than two years, so I don't know if we need to optimize automating updates for it.

I do think the version might make sense as a short commit SHA instead of 0_git<date>, but I don't feel that strongly either.

We don't really have any precedent for commit-versioned packages in wolfi it looks like (unless I missed one). We have some date-versioned packages, like wasi-libc.yaml which has `version: 0.20220525`.

Reply:
I totally agree with @imjasonh comments here, so, let me update the version of the pkg with the form that you mentioned 🤝